app/models/approval.rb

```diff
@@ -16,6 +16,7 @@ class Approval < ActiveRecord::Base
 
   scope :individual, -> { where(type: 'Approvals::Individual') }
 
+  validates :proposal, presence: true, allow_blank: true
```

Comment:
Probably...? Might need to create associated models in the fixtures by default or something.